src/xdocs/config_imports.xml

```diff
@@ -620,15 +620,26 @@ import android.*;
     </section>
 
     <section name="IllegalImport">
+      <p>Since Checkstyle 3.0</p>
       <subsection name="Description" id="IllegalImport_Description">
-        <p>Since Checkstyle 3.0</p>
         <p>
           Checks for imports from a set of illegal packages. By default, the
           check rejects all <code>sun.*</code> packages since
           programs that contain direct calls to the <code>sun.*</code> packages are <a
           href="https://www.oracle.com/technetwork/java/faq-sun-packages-142232.html">"not
           guaranteed to work on all Java-compatible platforms"</a>. To reject other packages,
-          set property <code> illegalPkgs</code> to a list of the illegal packages.
+          set property <code>illegalPkgs</code> to a list of the illegal packages.
+        </p>
+      </subsection>
+
+      <subsection name="Notes" id="IllegalImport_Notes">
+        <p>
+          Note that if variable <code>regexp</code> is set to <code>false</code>,
```

Comment:
`if property`
please recheck that in this doc there is no reference to "variable" term.

Reply:
done